### PR TITLE
Improve filter edition modal, add auto reply action

### PR DESCRIPTION
**Pull request by Swiip**
Partially fix https://github.com/ProtonMail/proton-mail-settings/issues/218

- Trash icon only shown for 2 or more conditions
- Add move to spam and trash
- Change message for `Move to inbox`
- Add auto reply action

### CODE REVIEW

File: containers/filters/editor/AutoReplyAction.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                        className="mr1"
+                        checked={activated}
+                        id="sendAutoReplyToggle"
+                        onChange={() => setActivated(!activated)}
+                    />
+                    <Label htmlFor="sendAutoReplyToggle">
+                        <span className="mr0-5">{c('Label').t`Send auto reply`}</span>
+                    </Label>
+                </Field>
+            </Row>{' '}
```
Comment (EpokK) [inline containers/filters/editor/AutoReplyAction.js]:
```suggestion
            </Row>
```

File: containers/filters/editor/AutoReplyAction.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Label, Field, Toggle, RichTextEditor } from 'react-components';
+import { noop } from 'proton-shared/lib/helpers/function';
+import Row from '../../../components/container/Row';
```
Comment (EpokK) [inline containers/filters/editor/AutoReplyAction.js]: Import `Row` from `react-components` please.

File: containers/filters/editor/AutoReplyAction.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Label, Field, Toggle, RichTextEditor } from 'react-components';
```
Comment (EpokK) [inline containers/filters/editor/AutoReplyAction.js]:
```suggestion
import { Label, Field, Toggle, RichTextEditor, Row } from 'react-components';
```

**Comment by emnproton** [comment]
Verified as fixed.

**Pull request merged**